tests/cypress/fixtures/deleteSite.groovy

```diff
@@ -0,0 +1,6 @@
+import org.jahia.services.sites.JahiaSitesService
```

Comment:
Don't think this script is needed as it looks like we're already using the one from `@jahia/cypress`